grails-app/assets/stylesheets/region-main.css

```diff
@@ -118,40 +118,48 @@ li.regionLink:hover {
     height: 400px;
 }
 
-#groups, #species {
+#groups,
+#species {
     margin-bottom: .8em;
 }
 
-#groups thead, #species thead {
+#groups thead,
+#species thead {
     height: 30px;
     border-bottom: 1px solid #dddddd;
 }
 
-#groups thead tr, #species thead tr {
+#groups thead tr,
+#species thead tr {
     position: relative;
     width: 100%;
 }
 
-#groups tbody, #species tbody {
+#groups tbody,
+#species tbody {
     height: 530px;
     max-height: 530px;
     overflow-y: auto;
 }
 
-#groups tbody td, #species tbody td {
+#groups tbody td,
+#species tbody td {
     border: 0;
 }
 
-#groups td, #species td {
+#groups td,
+#species td {
     padding: 1px 0.5em 1px 0;
 }
 
-#groups tbody .spinner, #species tbody .spinner {
+#groups tbody .spinner,
+#species tbody .spinner {
     display: block;
     height: 99.5%;
 }
 
-#groups tbody td.spinner i, #species tbody td.spinner i {
+#groups tbody td.spinner i,
+#species tbody td.spinner i {
```

Comment:
I'm assuming the i tag here is used for icon. Is it still i in this case or have we replaced it with span? Does it work?